ui-src/src/components/Chatbox.tsx

```diff
@@ -24,24 +24,22 @@ type ChatProps = {
 /////////////////////////////////////////////////////////////
 ///////////////////////////////////////////////////////////////
 
-class Chatbox extends React.Component<any, any> {
-
-  public render() {
-    const {currentGame} = this.props
-    if (currentGame === null) {
-      return <div/>
-    } else {
-      const {gameHash, chats} = currentGame
-      return(
-        <div className='chat-box'>
-          <MessagesList chats={chats} mode="blocks" />
-          <InputForm gameHash={gameHash} />
-        </div>
-      )
-    }
+export const ChatboxComponents = () => {
```

Comment:
While unconventional, I actually think the ChatboxComponents function is a clever integration of Flexbox for the HUD elements. 👍